docs/dev/table/connect.md

```diff
@@ -426,9 +404,32 @@ For more information about time handling in Flink and especially event-time, we
 
 In order to control the event-time behavior for tables, Flink provides predefined timestamp extractors and watermark strategies.
 
+Please refer to [CREATE TABLE statements](sql/create.html#create-table) for more information about defining time attributes in DDL.
+
 The following timestamp extractors are supported:
 
 <div class="codetabs" markdown="1">
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- use the existing TIMESTAMP(3) field in schema as the rowtime attribute
+CREATE TABLE MyTable (
+  ts_field TIMESTAMP(3),
+  WATERMARK FOR ts_field AS ...
+) WITH (
+  ...
+)
+
```

Comment:
Do we support `timestampsFromSource`?

Reply:
We don't support yet.

Reply:
Can we add a comment to tell user? Because the DDL is the first tab now.